libbeat/docs/dashboards.asciidoc

```diff
@@ -1,83 +1,63 @@
 [[load-kibana-dashboards]]
-=== Loading the Beats Dashboards
+=== Importing the Beats Dashboards
 
 Kibana has a large set of visualization types that you can combine to create
 the perfect dashboards for your needs. But this flexibility can be a bit
 overwhelming at the beginning, so we have created some sample dashboards to get you
 started and to demonstrate what is possible based on the Beats data.
 
-Each Beat comes packaged with a script that you can use to import the example dashboards,
+Each Beat comes packaged with the `scripts/import_dashboards` script that you can use to import the example dashboards,
 visualizations, and searches for that Beat. The script also creates index patterns, such as 
 `packetbeat-*` for Packetbeat, `metricbeat-*` for Metricbeat, and so on. 
 
-Use the following commands to run the script:
+
+The command line options of the script, together with more examples, are available under the
+<<import-dashboards,import_dashboards>> section.
+
+Use the following command to import the Kibana dashboards for any of the Elastic Beats:
 
 *deb, rpm, and mac:*
 
-From the directory where you installed the Beat, change to the `kibana` directory and run the `import_dashboards.sh` script. For example:
+From the directory where you installed the Beat, run the `import_dashboards` script. For example, to import the
+Metricbeat dashboards, and the index pattern, use:
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-cd kibana/ 
-./import_dashboards.sh
+$ scripts/import_dashboards -beat metricbeat
 ----------------------------------------------------------------------
 
-On deb and rpm, the `kibana` folder is located under the home path, which is `/usr/share/beat_name/` unless you change it.
+On deb and rpm, the `scripts` folder is located under the home path, which is `/usr/share/beat_name/` unless you change it.
```

Comment:
I suggest to replace `beat_name` with `{beat_name}` as we do it similarly for other path examples. This makes it more obvious that this part is a variable that has to be replaced.